SF iOS/SF iOS/Models/EventDataSource.h

```diff
@@ -31,6 +32,6 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithEventType:(EventType)eventType;
 - (void)refresh;
 - (Event *)eventAtIndex:(NSUInteger)index;
-
+- (void)filterEventsWith:(NSString *)searchTerm;
```

Comment:
this method name feels more Swift-y than ObjC-y (maybe call it `filterEventsWithSearchTerm:`?)